client/src/apps/match/MatchApp.tsx

```diff
@@ -129,9 +127,9 @@ function MatchApp() {
         setCountHistory([...countHistory, count]);
         setCount(newCount);
     };
-    const handleCount = (key: countKeys) => {
-        handleSetCount({ ...count, [key]: count[key] + 1 });
-    };
+    //const handleCount = (key: countKeys) => {
+    //    handleSetCount({ ...count, [key]: count[key] + 1 });
+    //};

```

Comment:
It should be safe to remove this - if we wanted to have access to the more general version of the function you're using below, I think we can check back in commit history to grab it :smiley:  worst case, the code you wrote is still pretty close to this!